pynailgun/ng.py

```diff
@@ -1121,14 +1121,13 @@ def main():
             except CalledProcessError as e:
                 # Works in systems such as Mac OS or Nix that in which blp-server is a script
                 try:
-                    jvm_options_with_prefix = [ "-J" + opt for opt in jvm_options_no_prefix ]
```

Comment:
You can't really remove this because `jvm_options_with_no_prefix` accumulates all Java options coming from either `cmd_args` or `jvm_options_from_file`. Maybe the best thing to do here would be to add every arg in `cmd_args` that is not a jvm option to an array `client_args` and then use that to sys process instead of removing `jvm_options_with_prefix`